src/victory-container/victory-container.js

```diff
@@ -79,6 +107,7 @@ export default class VictoryContainer extends React.Component {
         <title id="title">{this.props.title}</title>
         <desc id="desc">{this.props.desc}</desc>
         {this.props.children}
+        {React.cloneElement(this.props.portalComponent, {ref: "portal"})}
```

Comment:
(i still end up always reaching for the string refs for some reason, will replace it)
naming collision with what?